vault/resource_aws_auth_backend_roletag_blacklist.go

```diff
@@ -0,0 +1,152 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/vault/api"
+)
+
+var (
+	awsAuthBackendRoleTagBlacklistBackendFromPathRegex = regexp.MustCompile("^auth/(.+)/config/tidy/roletag-blacklist$")
+)
+
+func awsAuthBackendRoleTagBlacklistResource() *schema.Resource {
+	return &schema.Resource{
+		Create: awsAuthBackendRoleTagBlacklistWrite,
+		Read:   awsAuthBackendRoleTagBlacklistRead,
+		Update: awsAuthBackendRoleTagBlacklistWrite,
+		Delete: awsAuthBackendRoleTagBlacklistDelete,
+		Exists: awsAuthBackendRoleTagBlacklistExists,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"backend": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Unique name of the auth backend to configure.",
+				ForceNew:    true,
+				// standardise on no beginning or trailing slashes
```

Comment:
Won't this require a `DiffSuppressFunc` as well if slashes are included? I don't see a configuration that explicitly tests this condition, but I think it will cause a perpetual diff.

Reply:
StateFunc should make DiffSuppressFunc unnecessary:

> // StateFunc is a function called to change the value of this before
>    // storing it in the state (and likewise before comparing for diffs).
>    // The use for this is for example with large strings, you may want
>    // to simply store the hash of it.